packages/@aws-cdk/aws-stepfunctions-tasks/lib/start-execution.ts

```diff
@@ -0,0 +1,110 @@
+import iam = require('@aws-cdk/aws-iam');
+import sfn = require('@aws-cdk/aws-stepfunctions');
+import { FieldUtils } from '@aws-cdk/aws-stepfunctions/lib/fields';
+import { Stack } from '@aws-cdk/core';
+
+/**
+ * Properties for ExecuteStateMachineProps
+ */
+export interface ExecuteStateMachineProps {
+  /**
+   * The JSON input for the execution, same as that of StartExecution.
+   * See {@link https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartExecution.html|StartExecution}
+   */
+  readonly input?: { [key: string]: any };
+
+  /**
+   * The name of the execution, same as that of StartExecution.
+   * See {@link https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartExecution.html|StartExecution}
+   */
+  readonly name?: string;
+
+  /**
+   * Pause the workflow until the task token is returned by SendTaskSuccess or SendTaskFailure.
+   * See {@link https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html#connect-wait-token|WaitForCallback}
+   */
+  readonly waitForTaskToken?: boolean;
+
+  /**
+   * Wait for the task/job to complete before processing the next state.
+   * See {@link https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html#connect-sync|Sync}
```

Comment:
Newline between these lines in the comment please.

Also, write link as `@see https://...`.